server/builder.js

```diff
@@ -66,24 +67,26 @@ async function processPush(push) {
 	}
 
 	// update node_modules
-	await cmd('npm install');
+	await cmd('yarn install');
```

Comment:
BTW http://blog.npmjs.org/post/171556855892/introducing-npm-ci-for-faster-more-reliable should be useful here.